src/MPAGSCipher/CipherMode.hpp

```diff
@@ -0,0 +1,9 @@
+#ifndef MPAGSCIPHER_CIPHERMODE_HPP
+#define MPAGSCIPHER_CIPHERMODE_HPP
+
+enum class CipherMode{
+    encrypt,
+    decrypt
```

Comment:
Should be UpperCamelCase according to our naming convention:
```suggestion
    Encrypt,
    Decrypt
```